apps/website/src/examples/ExpandableBlock.form.tsx

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { ExpandableBlock, Label, Input, InputGroup, Radio } from '@itwin/itwinui-react';
+
+export default () => {
+  const nameSection = (
+    <>
+      <Label required>Name</Label>
```

Comment:
these labels need to be connected with the input using `htmlFor` and `id`

Reply:
Fixed